pkg/testrunner/result/output.go

```diff
@@ -59,7 +59,7 @@ func Output(config *Config, tmKubeconfigPath, namespace string, tr *tmv1beta1.Te
 		return err
 	}
 
-	osConfig, err := getOSConfig(tmKubeconfigPath, s3Endpoint, namespace)
+	osConfig, err := getOSConfig(tmKubeconfigPath, s3Endpoint, namespace, config.S3SSL)
```

Comment:
you could inline `s3Endpoint` to `config.S3Endpoint` and remove the var on line 43

Reply:
done